src/scss/layout/_header.scss

```diff
@@ -1,6 +1,107 @@
 body{
-    background-color: rgb(85, 175, 202);
+  background-color: rgb(85, 175, 202);
```

Comment:
space before braces `body {`
please do a variable for the background color of the body